CI failing due to upgrading brew (`Could not symlink bin/idle3`)

https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090

```
+ rm -f /usr/local/bin/2to3
+ brew update
==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations

Updated 6 taps (homebrew/core, homebrew/cask, homebrew/bundle, homebrew/services, mongodb/brew and aws/tap).
==> New Formulae
cloudflare-wrangler2
httm
keploy
libfyaml
nap
txt2man
==> New Casks
duplicacy-cli
scene-maestro
scenica-player
scrolla
unraid-usb-creator
vero
wooshy
==> Outdated Formulae
aws-sam-cli
imagemagick
libmpc
libraw
mongodb-community@5.0
python@3.10
yq
==> Outdated Casks
virtualbox

You have 7 outdated formulae and 1 outdated cask installed.
You can upgrade them with brew upgrade
or list them with brew outdated.

real	0m20.168s
user	0m8.825s
sys	0m6.756s
+ brew uninstall --ignore-dependencies lima
Uninstalling /usr/local/Cellar/lima/0.14.1... (72 files, 37.9MB)

real	0m3.171s
user	0m1.198s
sys	0m0.694s
+ brew install qemu bash coreutils curl jq
Warning: qemu 7.2.0 is already installed and up-to-date.
To reinstall 7.2.0, run:
  brew reinstall qemu
==> Fetching bash
Warning: curl 7.86.0 is already installed and up-to-date.
==> Downloading https://ghcr.io/v2/homebrew/core/bash/manifests/5.2.[15](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:16)
To reinstall 7.86.0, run:
  brew reinstall curl
Warning: jq 1.6 is already installed and up-to-date.
To reinstall 1.6, run:
  brew reinstall jq
==> Downloading https://ghcr.io/v2/homebrew/core/bash/blobs/sha256:05a5f9435c9e9ffe8377b03e0ca6b27bbb32cc01aff47dd[16](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:17)92cd8d7e735ab3a
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:05a5f9435c9e9ffe8377b03e0ca6b27bbb32cc01aff47dd1692cd8d7e735ab3a?se=2022-12-20T01%3A15%3A00Z&sig=UOZU2w%2Fgxe6tOrXauE0qSFE%2Bs%2FtQ2RgzVZWLrTGTbV0%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Fetching coreutils
==> Downloading https://ghcr.io/v2/homebrew/core/coreutils/manifests/9.1
==> Downloading https://ghcr.io/v2/homebrew/core/coreutils/blobs/sha256:7c9f988b4f9207415a5c96efd32376bc8cf2b280a7a36fbebb0b8fc334a14056
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:7c9f988b4f9207415a5c96efd32376bc8cf2b280a7a36fbebb0b8fc334a14056?se=2022-12-20T01%3A15%3A00Z&sig=M9vXGen%2FFzNK2dFYHZolNM1LX7PPH7A0haupn0lowYg%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Pouring bash--5.2.15.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/bash/5.2.15: 162 files, 11.7MB
==> Running `brew cleanup bash`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Pouring coreutils--9.1.monterey.bottle.tar.gz
==> Caveats
Commands also provided by macOS and the commands dir, dircolors, vdir have been installed with the prefix "g".
If you need to use these commands with their normal names, you can add a "gnubin" directory to your PATH with:
  PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
==> Summary
🍺  /usr/local/Cellar/coreutils/9.1: 480 files, 13.8MB
==> Running `brew cleanup coreutils`...
==> Caveats
==> coreutils
Commands also provided by macOS and the commands dir, dircolors, vdir have been installed with the prefix "g".

If you need to use these commands with their normal names, you can add a "gnubin" directory to your PATH with:
real	0m19.303s
  PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
user	0m7.872s
sys	0m4.477s
+ brew upgrade
==> Upgrading 7 outdated packages:
aws/tap/aws-sam-cli 1.66.0 -> 1.67.0
yq 4.30.5 -> 4.30.6
libraw 0.20.2_4 -> 0.21.0
python@3.10 3.10.8 -> 3.10.9
mongodb/brew/mongodb-community@5.0 5.0.11 -> 5.0.14
imagemagick 7.1.0-54 -> 7.1.0-55
libmpc 1.2.1 -> 1.3.1
==> Fetching mongodb/brew/mongodb-community@5.0
==> Downloading https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-5.0.14.tgz
==> Fetching aws/tap/aws-sam-cli
==> Downloading https://github.com/aws/aws-sam-cli/releases/download/v1.67.0//aws-sam-cli-1.67.0.sierra.bottle.tar.gz
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/92205085/8ab80673-8e4a-4bde-ae41-5ce104ad3c79?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20221220%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20221220T010947Z&X-Amz-Expires=300&X-Amz-Signature=d6996567bf8f37be90a6c2c9900fab3d434d2f45c7bc914e0af4031ca5eecd7b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=92205085&response-content-disposition=attachment%3B%20filename%3Daws-sam-cli-1.67.0.sierra.bottle.tar.gz&response-content-type=application%2Foctet-stream
==> Fetching yq
==> Downloading https://ghcr.io/v2/homebrew/core/yq/manifests/4.30.6
==> Downloading https://ghcr.io/v2/homebrew/core/yq/blobs/sha256:6b5e98258fb5938d56049651f82f4b18cf85efc8a28d[17](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:18)b8523855d6421ee7a8
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:6b5e98258fb5938d56049651f82f4b[18](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:19)cf85efc8a28d17b8523855d6421ee7a8?se=2022-12-20T01%3A15%3A00Z&sig=S4OYolHwn4JVPuaZ4rzf5kz6wILqwc3yfjwwviUSako%3D&sp=r&spr=https&sr=b&sv=20[19](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:20)-12-12
==> Fetching libraw
==> Downloading https://ghcr.io/v2/homebrew/core/libraw/manifests/0.21.0
==> Downloading https://ghcr.io/v2/homebrew/core/libraw/blobs/sha256:7dd1a73ccdddcde857cb3d88e55112b3c249414a85eb8e35dbd9dcf2dbe88360
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:7dd1a73ccdddcde857cb3d88e55112b3c249414a85eb8e35dbd9dcf2dbe88360?se=[20](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:21)22-12-20T01%3A15%3A00Z&sig=pEMIGYqUZsEo5Wc9QeE6bcXBdNMplUy%2BsL2rOOUo1CQ%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Fetching python@3.10
==> Downloading https://ghcr.io/v2/homebrew/core/python/3.10/manifests/3.10.9
==> Downloading https://ghcr.io/v2/homebrew/core/python/3.10/blobs/sha256:7e95777f8c614e01f6c94419cf7feb72aa34a7940173a5b04f359ac845b28509
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:7e95777f8c614e01f6c94419cf7feb72aa34a7940173a5b04f359ac845b28509?se=2022-12-20T01%3A15%3A00Z&sig=DazXexesRQaPCQY47GiBxfk1pEJuH1u9Aa8i7BLnwVI%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Fetching dependencies for imagemagick: libraw
==> Fetching imagemagick
==> Downloading https://ghcr.io/v2/homebrew/core/imagemagick/manifests/7.1.0-55
==> Downloading https://ghcr.io/v2/homebrew/core/imagemagick/blobs/sha256:d886aedda4cb79363cb01da7dcd57e858ab6db0e683bd49f9d42bb51e01f7715
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:d886aedda4cb79363cb01da7dcd57e858ab6db0e683bd49f9d42bb51e01f7715?se=2022-12-20T01%3A15%3A00Z&sig=cC1t57GY8fz7b7hLXH3CD92nApkSa0SznXIR6eX%2FHgo%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Fetching libmpc
==> Downloading https://ghcr.io/v2/homebrew/core/libmpc/manifests/1.3.1
==> Downloading https://ghcr.io/v2/homebrew/core/libmpc/blobs/sha256:c32f2c3fe7ab06e308e6fa74874e1d4d92ff6eb3598da6e0f8e6fa7a333350f5
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:c32f2c3fe7ab06e308e6fa74874e1d4d92ff6eb3598da6e0f8e6fa7a333350f5?se=2022-12-20T01%3A15%3A00Z&sig=q2a2Njol92IQUSKAHr9rhPne5FwdO9M8T5Q5o8bXOiI%3D&sp=r&spr=https&sr=b&sv=2019-12-12
==> Upgrading mongodb/brew/mongodb-community@5.0
  5.0.11 -> 5.0.14 

==> Caveats
mongodb-community@5.0 is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

If you need to have mongodb-community@5.0 first in your PATH, run:
  echo 'export PATH="/usr/local/opt/mongodb-community@5.0/bin:$PATH"' >> /Users/runner/.bash_profile


To start mongodb/brew/mongodb-community@5.0 now and restart at login:
  brew services start mongodb/brew/mongodb-community@5.0
Or, if you don't want/need a background service you can just run:
  mongod --config /usr/local/etc/mongod.conf
==> Summary
🍺  /usr/local/Cellar/mongodb-community@5.0/5.0.14: 11 files, 183.4MB, built in 9 seconds
==> Running `brew cleanup mongodb-community@5.0`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
Removing: /usr/local/Cellar/mongodb-community@5.0/5.0.11... (11 files, 183.2MB)
Removing: /Users/runner/Library/Caches/Homebrew/mongodb-community@5.0--5.0.11.tgz... (65MB)
==> Upgrading aws/tap/aws-sam-cli
  1.66.0 -> 1.67.0 

==> Pouring aws-sam-cli-1.67.0.sierra.bottle.tar.gz
🍺  /usr/local/Cellar/aws-sam-cli/1.67.0: 5,141 files, 117.7MB
==> Running `brew cleanup aws-sam-cli`...
Removing: /usr/local/Cellar/aws-sam-cli/1.66.0... (5,082 files, 115.6MB)
Removing: /Users/runner/Library/Caches/Homebrew/aws-sam-cli--1.66.0.sierra.bottle.tar.gz... (23.7MB)
==> Upgrading yq
  4.30.5 -> 4.30.6 

==> Pouring yq--4.30.6.monterey.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> Summary
🍺  /usr/local/Cellar/yq/4.30.6: 9 files, 9.2MB
==> Running `brew cleanup yq`...
Removing: /usr/local/Cellar/yq/4.30.5... (9 files, 9.2MB)
Removing: /Users/runner/Library/Caches/Homebrew/yq--4.30.5... (3.6MB)
==> Upgrading libraw
  0.20.2_4 -> 0.[21](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:22).0 

==> Pouring libraw--0.21.0.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/libraw/0.21.0: 73 files, 5.7MB
==> Running `brew cleanup libraw`...
Removing: /usr/local/Cellar/libraw/0.20.2_4... (73 files, 5.6MB)
Removing: /Users/runner/Library/Caches/Homebrew/libraw--0.20.2_4... (1.6MB)
==> Upgrading python@3.10
  3.10.8 -> 3.10.9 

==> Pouring python@3.10--3.10.9.monterey.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/idle3
Target /usr/local/bin/idle3
already exists. You may want to remove it:
  rm '/usr/local/bin/idle3'

To force the link and overwrite all conflicting files:
  brew link --overwrite python@3.10

To list all files that would be deleted:
  brew link --overwrite --dry-run python@3.10

Possible conflicting files are:
/usr/local/bin/idle3 -> /Library/Frameworks/Python.framework/Versions/3.11/bin/idle3
/usr/local/bin/pydoc3 -> /Library/Frameworks/Python.framework/Versions/3.11/bin/pydoc3
/usr/local/bin/python3 -> /Library/Frameworks/Python.framework/Versions/3.11/bin/python3
/usr/local/bin/python3-config -> /Library/Frameworks/Python.framework/Versions/3.11/bin/python3-config
==> /usr/local/Cellar/python@3.10/3.10.9/bin/python3.10 -m ensurepip
==> /usr/local/Cellar/python@3.10/3.10.9/bin/python3.10 -m pip install -v --no-d
==> Caveats
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python@3.10/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.10/site-packages

tkinter is no longer included with this formula, but it is available separately:
  brew install python-tk@3.10

See: https://docs.brew.sh/Homebrew-and-Python
==> Summary
🍺  /usr/local/Cellar/python@3.10/3.10.9: 3,108 files, 56.4MB
==> Running `brew cleanup python@3.10`...
Removing: /usr/local/Cellar/python@3.10/3.10.8... (3,136 files, 56.9MB)
Removing: /Users/runner/Library/Caches/Homebrew/python@3.10--3.10.8... (14.6MB)
==> Upgrading imagemagick
  7.1.0-54 -> 7.1.0-55 

==> Pouring imagemagick--7.1.0-55.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/imagemagick/7.1.0-55: 807 files, 28.7MB
==> Running `brew cleanup imagemagick`...
Removing: /usr/local/Cellar/imagemagick/7.1.0-54... (807 files, 28.7MB)
Removing: /Users/runner/Library/Caches/Homebrew/imagemagick--7.1.0-54... (10.2MB)
==> Upgrading libmpc
  1.2.1 -> 1.3.1 

==> Pouring libmpc--1.3.1.monterey.bottle.tar.gz
🍺  /usr/local/Cellar/libmpc/1.3.1: 12 files, 464KB
==> Running `brew cleanup libmpc`...
Removing: /usr/local/Cellar/libmpc/1.2.1... (12 files, 407.4KB)
Removing: /Users/runner/Library/Caches/Homebrew/libmpc--1.2.1... (124.5KB)
==> Casks with 'auto_updates true' or 'version :latest' will not be upgraded; pass `--greedy` to upgrade them.
==> Upgrading 1 outdated package:
virtualbox 6.1.38,153438 -> 7.0.4,154605
==> Upgrading virtualbox
==> Downloading https://download.virtualbox.org/virtualbox/7.0.4/VirtualBox-7.0.4-154605-OSX.dmg
==> Running uninstall script VirtualBox_Uninstall.tool

Welcome to the VirtualBox uninstaller script.

Executing: /usr/bin/kmutil showloaded --list-only --bundle-identifier org.virtualbox.kext.VBoxUSB
No variant specified, falling back to release
Executing: /usr/bin/kmutil showloaded --list-only --bundle-identifier org.virtualbox.kext.VBoxNetFlt
No variant specified, falling back to release
Executing: /usr/bin/kmutil showloaded --list-only --bundle-identifier org.virtualbox.kext.VBoxNetAdp
No variant specified, falling back to release
Executing: /usr/bin/kmutil showloaded --list-only --bundle-identifier org.virtualbox.kext.VBoxDrv
No variant specified, falling back to release
The following files and directories (bundles) will be removed:
    /usr/local/bin/VirtualBox
    /usr/local/bin/VirtualBoxVM
    /usr/local/bin/VBoxManage
    /usr/local/bin/VBoxVRDP
    /usr/local/bin/VBoxHeadless
    /usr/local/bin/vboxwebsrv
    /usr/local/bin/VBoxBugReport
    /usr/local/bin/VBoxBalloonCtrl
    /usr/local/bin/VBoxAutostart
    /usr/local/bin/VBoxDTrace
    /usr/local/bin/vbox-img
    /Library/LaunchDaemons/org.virtualbox.startup.plist
    /Library/Application Support/VirtualBox/LaunchDaemons/
    /Library/Application Support/VirtualBox/VBoxDrv.kext/
    /Library/Application Support/VirtualBox/VBoxUSB.kext/
    /Library/Application Support/VirtualBox/VBoxNetFlt.kext/
    /Library/Application Support/VirtualBox/VBoxNetAdp.kext/
    /Applications/VirtualBox.app/

And the following KEXTs will be unloaded:
    org.virtualbox.kext.VBoxUSB
    org.virtualbox.kext.VBoxNetFlt
    org.virtualbox.kext.VBoxNetAdp
    org.virtualbox.kext.VBoxDrv

And the traces of following packages will be removed:
    org.virtualbox.pkg.vboxkexts
    org.virtualbox.pkg.virtualbox
    org.virtualbox.pkg.virtualboxcli

The uninstallation processes requires administrative privileges
because some of the installed files cannot be removed by a normal
user. You may be prompted for your password now...

unloading org.virtualbox.kext.VBoxUSB
Executing: /usr/bin/kmutil unload -b org.virtualbox.kext.VBoxUSB
unloading org.virtualbox.kext.VBoxNetFlt
Executing: /usr/bin/kmutil unload -b org.virtualbox.kext.VBoxNetFlt
unloading org.virtualbox.kext.VBoxNetAdp
Executing: /usr/bin/kmutil unload -b org.virtualbox.kext.VBoxNetAdp
unloading org.virtualbox.kext.VBoxDrv
Executing: /usr/bin/kmutil unload -b org.virtualbox.kext.VBoxDrv
Successfully unloaded VirtualBox kernel extensions.
Forgot package 'org.virtualbox.pkg.vboxkexts' on '/'.
Forgot package 'org.virtualbox.pkg.virtualbox' on '/'.
Forgot package 'org.virtualbox.pkg.virtualboxcli' on '/'.
Done.
==> Uninstalling packages; your password may be necessary:
==> Removing files:
/usr/local/bin/vboximg-mount
==> Running installer for virtualbox; your password may be necessary.
Package installers may write to any location; options such as `--appdir` are ignored.
installer: Package name is Oracle VM VirtualBox
installer: choices changes file '/private/tmp/choices20[22](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:23)1220-11619-15zkrfh.xml' applied
installer: Installing at base path /
installer: The install was successful.
==> Changing ownership of paths required by virtualbox; your password may be necessary.
==> Purging files for version 6.1.38,15[34](https://github.com/lima-vm/lima/actions/runs/3736517443/jobs/6340903090#step:8:35)38 of Cask virtualbox
🍺  virtualbox was successfully upgraded!
==> Caveats
==> mongodb-community@5.0
mongodb-community@5.0 is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

If you need to have mongodb-community@5.0 first in your PATH, run:
  echo 'export PATH="/usr/local/opt/mongodb-community@5.0/bin:$PATH"' >> /Users/runner/.bash_profile


To start mongodb/brew/mongodb-community@5.0 now and restart at login:
  brew services start mongodb/brew/mongodb-community@5.0
Or, if you don't want/need a background service you can just run:
  mongod --config /usr/local/etc/mongod.conf
==> yq
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> python@3.10
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python@3.10/libexec/bin

You can install Python packages with
  pip3 install <package>
They will install into the site-package directory
  /usr/local/lib/python3.10/site-packages

tkinter is no longer included with this formula, but it is available separately:
  brew install python-tk@3.10

See: https://docs.brew.sh/Homebrew-and-Python
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -97,16 +97,9 @@ jobs:
       # jq:        required by test-example.sh to determine download URL for nerdctl
       run: |
         set -x
-        # Github runners seem to symlink to python2.7 version of 2to3,
-        # breaking upgrades to latest python@3.9
-        rm -f /usr/local/bin/2to3
-        time brew update
         # Github runners seem to have lima installed by brew already; we don't want/need it
-        # It also does throw an error during `brew upgrade`.
-        # Needs --ignore-dependencies because it is installed as a prerequisite for colima.
-        time brew uninstall --ignore-dependencies lima
+        time brew uninstall --ignore-dependencies lima colima
         time brew install qemu bash coreutils curl jq
-        time brew upgrade
     - name: Cache ~/Library/Caches/lima/download
       uses: actions/cache@v3
       with:

--- a/examples/archlinux.yaml
+++ b/examples/archlinux.yaml
@@ -1,9 +1,9 @@
 # This example requires Lima v0.7.0 or later
 images:
 # Try to use yyyyMMdd.REV image if available. Note that yyyyMMdd.REV will be removed after several months.
-- location: "https://geo.mirror.pkgbuild.com/images/v20221201.106936/Arch-Linux-x86_64-cloudimg-20221201.106936.qcow2"
+- location: "https://geo.mirror.pkgbuild.com/images/v20221215.111177/Arch-Linux-x86_64-cloudimg-20221215.111177.qcow2"
   arch: "x86_64"
-  digest: "sha256:f380f983484014c5237e446550d49f88122b8176187d78bfdcdb3bc5bfeecce7"
+  digest: "sha256:e03bca357b6962eb65832dba91bd3515ebd6fe6592748dd0e136502b39d88041"
 - location: "https://github.com/mcginty/arch-boxes/releases/download/v20220323/Arch-Linux-aarch64-cloudimg-20220323.0.qcow2"
   arch: "aarch64"
   digest: "sha512:27524910bf41cb9b3223c8749c6e67fd2f2fdb8b70d40648708e64d6b03c0b4a01b3c5e72d51fefd3e0c3f58487dbb400a79ca378cde2da341a3a19873612be8"

